index.html

```diff
@@ -2,110 +2,567 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <!-- <link rel="stylesheet" href="style.css"> --> <!-- uncomment styles --> 
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Binabox</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap" rel="stylesheet"/>
+    <link href="https://fonts.googleapis.com/css2?family=Bakbak+One:wght@400;500;600&display=swap" rel="stylesheet"/>
+    <link rel="stylesheet" href="normalize.css">
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <div id="container" class="container">
-        <div class="profile-folder">
-          <div class="folder">
-            <div class="title">
-              Yaba Kawaii Studio
-              <span class="title-btn">
-                <span class="btn1"></span>
-                <span class="btn2"></span>
-                <span class="btn3"></span>
-              </span>
+    <header class="header">
+        <h1 class="logo">
+            <img src="images/logo-white.svg" width="188" height="42" class="logo__image" alt="Binabox">
+        </h1>
+        <nav class="header-nav">
+            <ul class="header-nav__wrapper">
+                <li class="header-nav__item header-nav__item--active">
+                    <a href="#">Home</a>
+                </li>
+                <li class="header-nav__item">
+                    <a href="#">Blog</a>
+                </li>
+                <li class="header-nav__item">
+                    <a href="#">Catalog</a>
+                </li>
+                <li class="header-nav__item">
+                    <a href="#">Contact Us</a>
+                </li>
+            </ul>
+        </nav>
+        <div class="header-actions">
+            <form class="search-form" action="/search">
+                <button type="submit" class="search-form__btn">
+                    <img src="images/search.svg" width="24" height="24" alt="search">
```

Comment:
Please replace for all icons `<img>` tag to inline `<svg>` 
```
<svg class="btn__icon" width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg"> 
                            <path d="M19 19L14.65 14.65M17 9C17 13.4183 13.4183 17 9 17C4.58172 17 1 13.4183 1 9C1 4.58172 4.58172 1 9 1C13.4183 1 17 4.58172 17 9Z" stroke="#F5F8FB" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/> 
                        </svg>
```

Reply:
done